### ISSUE TITLE
Issue: Reason: 'Cannot find module \'iltorb\''

### ISSUE THREAD

**Issue by jo9553** [iann0036/amazon-connect-cfn#5]
The error is generated when deploying the example_stack.yaml to CloudFormation. Thanks.

**Comment by mariandumitrascu** [comment]
I'm having the same issue.

**Comment by jo9553** [comment]
I solved this error after I installed the npm packages locally and then uploaded to Lambda again. But still have other issues when trying to create resources. "Failed to create resource. waiting for selector `div.launch-page-login-link` failed", may need to adjust the timeout setting or other crawling problems.

**Comment by mariandumitrascu** [comment]
Yes. I did "npm i aws-sdk chrome-aws-lambda puppeteer-core request-promise request" in /lambda folder and packed it into a zip and used as the package for the lambda function and it worked.  It worked only for "AWS::Connect::Instance" though.  
Amazon changed slightly the interface for Connect and thats why I think "AWS::Connect::ContactFlow" is failing.  
But that can be fixed.

**Comment by iann0036** [comment]
Hey all,

Thanks for reporting. I don't have a whole bunch of time to work on this right now, but if you manage to figure out the fix, I'll happily merge the PR and re-upload the hosted ZIP.

It's also probably worth looking into the official API for AWS::Connect::ContactFlow now that it exists. Couldn't figure out the syntax for the call though.

**Comment by Caoimhin89** [comment]
@mariandumitrascu Were you able to figure out how to prevent AWS::Connect::ContactFlow from failing?

**Comment by Caoimhin89** [comment]
@iann0036 forked it. I'm working on a fix now. I'll send over a PR as soon as I get it all worked out. Awesome work, by the way!
